packages/browser-extension/src/scripts/background.js

```diff
@@ -64,10 +65,11 @@ const refreshData = (callback) => {
  */
 const report = async (message, tab, callback) => {
   try {
+    const explanation = message.explanation;
     const reason = message.reason;
     const userAgent = message.userAgent;
     const version = browser.runtime.getManifest().version;
-    const body = JSON.stringify({ reason, url: tab.url, userAgent, version });
+    const body = JSON.stringify({ reason, url: tab.url, userAgent, version, explanation });
```

Comment:
Please, order the properties alphabetically inside the object in the `JSON.stringify`.

Reply:
done